app/pages/project/instances/instance/tabs/NetworkingTab.tsx

```diff
@@ -133,6 +134,21 @@ const staticCols = [
     header: 'subnet',
     cell: (info) => <SubnetNameFromId value={info.getValue()} />,
   }),
+  colHelper.accessor('transitIps', {
+    header: 'Transit IPs',
+    cell: (info) => {
+      const transitIps = info.getValue()
+      return transitIps?.length ? (
+        <ListPlusCell numInCell={1}>
+          {transitIps.map((ip) => (
+            <div key={ip}>{ip}</div>
+          ))}
+        </ListPlusCell>
+      ) : (
+        <EmptyCell />
```

Comment:
This is a good idea — consider adding it to `ListPlusCell` if the existing call sites would benefit